### PR TITLE
Start or close multiple containers.

### PR DESCRIPTION
**Pull request by octoper**
This PR adds the ability to start or stop multiple containers at once.

### CODE REVIEW

**Comment by mattstauffer** [comment]
@octoper This is great! However, I just merged a PR that adds *some* of this, but not all. Would you mind updating your PR to reconcile with those new changes? Then I'd love to merge it!

**Comment by octoper** [comment]
I will create a new pull request with those changes!

**Comment by octoper** [comment]
Just update it!

File: app/Commands/StopCommand.php [hunk 1]
```diff
@@ -27,11 +26,19 @@ public function handle(Docker $docker, Environment $environment): void
...
-            $this->stop($container);
+        if (filled($containers)) {
+            foreach ($containers as $container) {
+                $this->stop($container);
+            }
+            return;
+        }
 
+        if ($this->option('all')) {
+            foreach (app(Docker::class)->stoppableTakeoutContainers() as $stopableContainer) {
```
Comment (mattstauffer) [inline app/Commands/StopCommand.php:39]: Just one small change; could you rename this variable to `stoppableContainer`?

Thanks!

Comment (delta1186) [reply app/Commands/StopCommand.php]: There is also an instance of docker already available 
$this->docker->
😁

Comment (octoper) [reply app/Commands/StopCommand.php]: Fixed! @mattstauffer

**Pull request merged**